app/models/manageiq/providers/vmware/infra_manager/refresh_parser.rb

```diff
@@ -233,8 +233,10 @@ def self.host_inv_to_ip(inv, hostname = nil)
       require 'ipaddr'
       default_gw = IPAddr.new(default_gw)
 
-      vnics   = inv.fetch_path("config", "network", "consoleVnic")
-      vnics ||= inv.fetch_path("config", "network", "vnic")
+      vnics         = inv.fetch_path("config", "network", "vnic")
+      console_vnics = inv.fetch_path("config", "network", "consoleVnic")
+      vnics.concat(console_vnics) unless console_vnics.nil?
```

Comment:
Yeah that's good too as long as we verify that the network section exists beforehand (I think so, but can you verify?).  I only thought we should tack on to_miq_a to both calls in case they come back nil (the VMware API is really screwy there and to_miq_a was introduced to make it all consistent).

```ruby
inv = inv.fetch_path('config', 'network')
vnics = inv['consoleVnic'].to_miq_a + inv['vnic'].to_miq_a
vnics.each do |vnic|
```